package.json

```diff
@@ -33,8 +34,8 @@
   },
   "devDependencies": {
     "dockerode": "^2.4.3",
-    "eslint": "^2.13.1",
-    "eslint-config-loopback": "^4.0.0",
+    "eslint": "^5.1.0",
+    "eslint-config-loopback": "^10.0.0",
```

Comment:
is there a reason you're using these versions instead of the latest, just curious?

Reply:
I wanted to match it with [another connector]( https://github.com/strongloop/loopback-connector-mongodb/commit/830f83a9eb59190975436b9d7bf353fe743b219b#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R40)

Reply:
It's best to upgrade to the latest version. Other connectors should be eventually upgraded too.